test/unit/data_validator_test.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# require 'test_helper'
+
+# class DataValidatorTest < ActiveSupport::TestCase
+#   include CqmValidators
+
+#   setup do
```

Comment:
recommend using skip() instead of commenting out file

Reply:
Seems reasonable. I'll switch to that.